src/test/java/net/snowflake/client/core/json/BytesConverterTest.java

```diff
@@ -0,0 +1,60 @@
+package net.snowflake.client.core.json;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.mockito.Mockito.mock;
+
+import java.math.BigInteger;
+import java.nio.ByteBuffer;
+import java.sql.Types;
+import net.snowflake.client.core.SFException;
+import net.snowflake.client.jdbc.SnowflakeResultSetSerializableV1;
+import org.apache.arrow.vector.Float8Vector;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BytesConverterTest {
```

Comment:
My proposal is to add test with exception, especially for DECIMAL

Reply:
I'm not sure about the reasons why we can't convert decimal to byte array. As for now my suggestion is to leave it as is.

Reply:
Can you create jira to investigate it later?

Reply:
SNOW-996569